packages/neptune-extension/src/labextension/utils/controller.js

```diff
@@ -0,0 +1,45 @@
+
+import { get } from 'lodash';
+
+export function createController(context, app) {
```

Comment:
It should be named Notebook and not Controller since its not meaning anything before you open this file. Also it should have methods:
getContent,
getMetadata,
save